include/common/msgType.h

```diff
@@ -350,6 +393,42 @@ struct ue_emm_info {
 };
 
 #define UE_EMM_INFO_BUF_SIZE sizeof(struct ue_emm_info)
+
+struct handover_request_Q_msg {
```

Comment:
Should add UE Security capabilities here?

Reply:
UE Security Capabilities is getting hardcoded in the s1ap layer. So skipped it from here.